src/sql/workbench/browser/modelComponents/splitviewContainer.component.ts

```diff
@@ -92,6 +99,25 @@ export default class SplitViewContainerImpl extends ContainerBase<FlexItemLayout
 		return basicView;
 	}
 
+	@debounce(300)
```

Comment:
300 seems a bit slow - there's a noticeable lag while dragging. It especially looks a little odd when there's other UI components that resize almost instantly. Would suggest seeing if lower values make it look smoother (as long as the perf overhead of resizing that much isn't too large)

Reply:
Changed it to 20, which was the lowest I saw other places using debounc. Looks a bit better now when resizing to bigger: 
![updatedDebounce](https://github.com/microsoft/azuredatastudio/assets/31145923/c2c7dbad-0d30-4ac5-8071-e99e725fad16)